mlflow/tracking/fluent.py

```diff
@@ -2046,6 +2046,34 @@ def get_logged_model(model_id: str) -> LoggedModel:
     return MlflowClient().get_logged_model(model_id)
 
 
+@experimental
+def last_logged_model() -> Optional[LoggedModel]:
```

Comment:
Maybe `get_last_logged_model`?

Reply:
Chose `last_logged_model` to align with `last_active_run`.